src/fhir/1_Questionnaire-C-IDAS.json

```diff
@@ -485,7 +485,7 @@
       "type": "choice",
       "required": false,
       "linkId": "cs-idas-7",
-      "text": "Loss of interest - less involved in usual activities (score only if change occurred acutely, i.e., in less than 1 month)",
+      "text": "In the last week have you observed loss of interest in your care recipient? (less involved in usual activities (score only if change occurred acutely, i.e., in less than 1 month))",
```

Comment:
@sierramatice just want to show you that there are parentheses within the parentheses here.  Perhaps, it was intended.  Just want to make sure:
![Screen Shot 2022-11-29 at 3 24 08 PM](https://user-images.githubusercontent.com/12942714/204670575-c6e16c12-7cbc-4e11-aefa-b2435ae86040.png)